src/screens/Analytics/GlobalSearch/GlobalSearchBarHelper.res

```diff
@@ -289,22 +340,18 @@ module FilterResultsComponent = {
               | Some(value) =>
                 value.options
                 ->Array.map(option => {
-                  <div
-                    className="flex justify-between hover:bg-gray-100 cursor-pointer hover:rounded-lg p-2 group items-center"
-                    onClick={_ => {
-                      let saparater =
-                        searchText->String.charAt(searchText->String.length - 1) == ":" ? "" : ":"
-                      setLocalSearchText(_ => `${searchText}${saparater}${option}`)
-                      setActiveFilter(_ => "")
-                    }}>
-                    <div className="bg-gray-200 py-1 px-2 rounded-md flex gap-1 items-center w-fit">
-                      <span className="font-medium text-sm">
-                        {`${value.categoryType
-                          ->getcategoryFromVariant
-                          ->String.toLocaleLowerCase} : ${option}`->React.string}
-                      </span>
-                    </div>
-                  </div>
+                  <FilterOption
+                    onClick={_ => option->onSuggestionClicked}
+                    value={`${value.categoryType
+                      ->getcategoryFromVariant
+                      ->String.toLocaleLowerCase} : ${option}`}
```

Comment:
is there any specific reason to use `toLocaleLowerCase` and not `toLowerCase` ?

Reply:
no we can use normal only will make this change